src/models/Prompts.ts

```diff
@@ -31,9 +32,10 @@ We would create a plan with 1 steps, #E1:
 #E1 would call the calculate tool with parameters ["+", "17", "5"] and describe the step as "Add 5 to the result"
 
 Example 2: if the user were to give the task: Create a graph to highlight my top 10 customers last year, my tables are Transactions, Users, and Products
-We would create a plan with 2 steps, #E1 and #E2:
-#E1 would call the getData tool with parameters ["Get the top 10 customers in terms of spent last week"] and describe the step as "Get the top 10 customers in terms of spent last week from the Transactions and Users data"
-#E2 would call the createChart tool with parameters ["#E1"] and describe the step as "Generate a chart to highlight the top 10 customers"
+We would create a plan with 3 steps, #E1, #E2 and #E3:
+#E1 would call the askHuman tool to ask the user what kind of graph want to create. Could be a bar graph or a doughnut graph.
+#E2 would call the getData tool with parameters ["Get the top 10 customers in terms of spent last week"] and describe the step as "Get the top 10 customers in terms of spent last week from the Transactions and Users data"
+#E3 would call the createChart tool with parameters ["#E2"] and describe the step as "Generate a chart to highlight the top 10 customers"
```

Comment:
Missing step 1 as a parameter to step 3
would call the createChart tool with parameters [generate a chart with data #E2 in format #E1] and describe the step as "Generate a chart to highlight the top 10 customers"